src/lib/db.ts

```
@@ -0,0 +1,32 @@
+import dotenv from "dotenv";
+import { drizzle } from "drizzle-orm/postgres-js";
+import postgres from "postgres";
+dotenv.config();
+const DATABASE_URL = `postgres://${process.env.API_POSTGRES_USER}:${process.env.API_POSTGRES_PASSWORD}@${process.env.API_POSTGRES_HOST}:${process.env.API_POSTGRES_PORT}/${process.env.API_POSTGRES_DATABASE}`;
+const client = postgres(DATABASE_URL, {
+	prepare: false,
```

Biome (1.9.4)

[error] This property value named prepare is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named prepare


(lint/suspicious/noDuplicateObjectKeys)